packages/cli/src/tools/installPods.js

```diff
@@ -40,14 +40,24 @@ async function installPods({
         },
       ]);
 
+      // This is only shown if it takes more than 30 seconds for the CocoaPods installation to finish
+      const cocoaPodsInstallationTimeMessage = setTimeout(
+        () => logger.info('Installing CocoaPods, this may take a few minutes'),
+        30000,
```

Comment:
can we extract this to a variable like `COCOAPODS_INSTALLATION_TIMEOUT` so it's easier to update in future?